detection-rules/attachment_microsoft_image_lure_qr_code.yml

```
@@ -17,6 +16,13 @@ source: |
               any(.scan.strings.strings, regex.icontains(.,  'scan|camera'))   
               and any(.scan.strings.strings, regex.icontains(.,  '\bQR\b|Q\.R\.|barcode'))
           )
+          or (
+              any(file.explode(.), .scan.qr.type == "url"
+                  // recipient email address is present in the URL, a common tactic used in credential phishing attacks 
+                  and any(recipients.to, strings.icontains(..scan.qr.data, .email.email)
+              )
```

```suggestion
                  and any(recipients.to, strings.icontains(..scan.qr.data, .email.email))
```
sorry messed this one up